gui/src/main/java/org/verapdf/cli/commands/VeraCliArgParser.java

```diff
@@ -91,9 +94,12 @@
     @Parameter(names = { PROFILES_WIKI_FLAG, PROFILES_WIKI }, description = "The path to the folder containing Validation Profiles wiki.")
     private String profilesWikiPath = "https://github.com/veraPDF/veraPDF-validation-profiles/wiki";
 
-    @Parameter(names = { REPORT_TYPE_FLAG, REPORT_TYPE}, description = "Operation to be performed", converter = ProcessingTypeConverter.class)
+    @Parameter(names = { REPORT_TYPE_FLAG, REPORT_TYPE}, description = "Operation to be performed.", converter = ProcessingTypeConverter.class)
     private ProcessingType processingType = ProcessingType.VALIDATING_AND_FEATURES;
 
+	@Parameter(names = {LOAD_CONFIG_FLAG, LOAD_CONFIG}, description = "Loads config form default file, all config flags are ignored.")
```

Comment:
1) We have to override all config fields if there is any more arguments passed (for example, if config contains showPassedRules false, but we set it as another argument as true, we should process with true value). Override it for processing, not for saved config file
2) Change that flag to Path/String value. If it is empty, then we will choose default config. In other way we will try to load the config from the given Path. 
3) Add a flag for saving new config with the same logic as described in point 2